assets/js/common/PatchList/PatchList.jsx

```diff
@@ -8,6 +8,10 @@ import { format as formatDate } from 'date-fns';
 import classNames from 'classnames';
 import { noop } from 'lodash';
 
+import {
+  createStringSortingPredicate,
+  createDateSortingPredicate,
+} from '@common/Table/sorting';
```

Comment:
Could you re-export these from `index.js` in `@common/Table` so we can do `import Table, {create...} from '@common/Table'`?